### PR TITLE
Res max 20201209 2

### PR DESCRIPTION
**Pull request by ZeroPointMax**
Ziehbegehren des heutigen Seminars.
Hier auch Prüfungsvorbereitung dabei

### CODE REVIEW

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/RES/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/DBS/README.md
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/PRO/README.md
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/SWE/README.md
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/NET/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- RES/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

File: RES/README.md [hunk 1]
```diff
@@ -710,3 +714,85 @@ quota --version # Kontrolle
...
+
+```bat
+@echo off
+net use E: /delete /yes
+net use F: /delete /yes
+net use O: /delete /yes
+
+net use E: \\192.168.71.100\Transfer\Glinka
+net use F: \\192.168.71.100\Transfer\Guenther
+net use O: \\192.168.71.100\Transfer\Herbst
```
Comment (sthinbetween) [inline RES/README.md:749]: "pause" noch, vielleicht?

File: RES/README.md [hunk 2]
```diff
@@ -710,3 +714,85 @@ quota --version # Kontrolle
...
+#!/bin/bash
+if [ -f "service.conf" ]; then
+  echo "Datei existiert schon. Fahre trotzdem fort."
+fi
+touch service.conf
+chmod 700 service.conf
+echo Zeile1 > service.conf
+echo Zeile2 >> service.conf
+sed -i 's/Zeile2/Zeile2=ON/g' service.conf
+```
```
Comment (sthinbetween) [inline RES/README.md:773]: cat service.conf für die Ausgabe noch

File: RES/pruefungsfragen.md [hunk 3]
```diff
@@ -93,3 +94,16 @@ for ((i = 1; i <= $#; i++ )); do printf '%s\n' "Arg $i: ${!i}" done
...
   - Kontingente: logische Speicher-Begrenzungen für Ordner
   - verhindert übermäßige Nutzung durch Programme und Nutzer, damit System weiter arbeiten kann
+
+Teil 2:
+
+- Browser zeigt "Unable to connect" --> was tun?
+  - physische Verbindung prüfen
+  - IP-Konfiguration prüfen
+    - Windows: ``ipconfig``
+    - Linux: ``ip a``
```
Comment (sthinbetween) [inline RES/pruefungsfragen.md:104]: oder ``ifconfig``

**Review by sthinbetween — APPROVED** [review]
Nur Kleinigkeiten ;) 

Danke!

File: RES/README.md [hunk 4]
```diff
@@ -710,3 +714,85 @@ quota --version # Kontrolle
...
+
+- Zweck: freie Speicherung von Daten auf priv. Server
+- Synchronisation von Daten
+- Videokonferenzen und Screenshare möglich (via Erweiterungen)
+- Anm. v. Max: FOSS-Ersatz für Cloudspeicher
+- Vorteile:
+  - kein Kontrollverlust über Daten
+- Nachteile:
+  - eigene Resourcen zum Betrieb nötig
+  - hoher Konfigurationsauwand <!--naja nee-->
```
Comment (RvNovae) [inline RES/README.md:728]:
```suggestion
  - hoher Konfigurationsaufwand <!--naja nee-->
```

**Review by RvNovae — APPROVED** [review]
Nur ein Typo ansonsten super, danke :)

File: RES/README.md (hunk 1 above)
Comment (ZeroPointMax) [reply RES/README.md:749]: Pausen sind für Noobs :P

File: RES/README.md (hunk 2 above)
Comment (ZeroPointMax) [reply RES/README.md:773]: Ups

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/RES/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/DBS/README.md
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/PRO/README.md
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/SWE/README.md
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/NET/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- RES/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

**Pull request merged**